### ISSUE TITLE
Custom keyboard shortcuts

### ISSUE THREAD

**Issue by bossanovaorca** [Swirt/typertools-src#20]
Is there support for customizing keyboard shortcuts in TyperTools?

**Comment by Swirt** [comment]
No, unfortunately. Photoshop doesn't allow you to listen to keypresses at all (when the extension panel is out of focus). I found a workaround, but it only works with 4 keys and not very well.

**Comment by bossanovaorca** [comment]
ah that's unfortunate, it would be nice to have configurable shortcuts or additional shortcuts for other actions like changing type size and styles. thanks for the response
